src/CoreShop/Bundle/ShippingBundle/DependencyInjection/Configuration.php

```diff
@@ -190,9 +190,11 @@ private function addPimcoreResourcesSection(ArrayNodeDefinition $node)
                             ->scalarNode('shipping')->defaultValue('/bundles/coreshopshipping/pimcore/css/shipping.css')->end()
                         ->end()
                     ->end()
-                    ->scalarNode('permissions')
-                        ->cannotBeOverwritten()
-                        ->defaultValue(['carrier', 'shipping_rule'])
+                    ->arrayNode('permissions')
```

Comment:
don't overwrite permission configurations, you cannot change them anyway and its not part of this PR